qemu/qmp.go

```diff
@@ -727,6 +727,49 @@ func (q *QMP) ExecuteBlockdevDel(ctx context.Context, blockdevID string) error {
 	return q.executeCommand(ctx, "x-blockdev-del", args, nil)
 }
 
+// ExecuteNetdevAdd adds a Net device to a QEMU instance
+// using the netdev_add command. netdevID is the id of the device to add.
+// Must be valid QMP identifier. Now it supports default tap created by kata, so use default 8 queues.
+func (q *QMP) ExecuteNetdevAdd(ctx context.Context, netdevID, ifname, downscript, script string) error {
+	args := map[string]interface{}{
+		"type":       "tap",
```

Comment:
should we make this configurable? (queues, type)

Reply:
Yes. queues should be configurable

Reply:
okay. eventually there'll be more fields needed for FDs and type, but I think as a starting point we can go with just tap.  Please make queues an argument?

Reply:
@amshinde ^^ WDYT?

Reply:
Agree. type and queues should be configurable definitely.

Reply:
Agreed with @egernst. At some point we will want vhost-user support as well but we can start with tap first. And it would be good to make `queues` an argument. We might want to expose it via SandboxConfig in kata containers.

Reply:
Updated.